### ISSUE TITLE
Feature suggestion: Encrypted email client?

### ISSUE THREAD

**Issue by ghost** [keybase/keybase-issues#888]
Zawinski's Law states that "Every program attempts to expand until it can read mail" - but it may seem to be useful for Keybase to be able to send messages (not necessarily to read them) from the cli client. If you add SMTP settings to Keybase, it would allow you to say for example:

`keybase message chris`

... and then it would bring up an interface where you can easily send an encrypted message to the email address associated with chris' account.

Similarly,

`keybase message chris -attach file.jpg`

... would send a message to chris' email with the file attached.

**Comment by compumike08** [comment]
I like the idea of an encrypted messaging protocol built into Keybase, however right now I'd be more interested in a built-in/integrated e-mail client if it could be done faster. The added hastle of having to encrypt a message in Keybase and copy and paste it into an e-mail, or having to copy an encrypted email into Keybase to decrypt it, is going to cut down on the number of people who are willing to use Keybase for encrypted messaging (e-mail or otherwise). I've talked to few friends who have tried Keybase after I gave them invitations, and they all said while they played around with Keybase for a few hours after installing it, they hadn't touched it since. They all liked the idea of Keybase, but for most people Keybase just doesn't have enough practical uses that are also user friendly. I realize that Keybase is still in the alpha stage, however I too was surprised when I first tired Keybase only to find that using it to send encrypted e-mails was so difficult, requiring so much copying and pasting. Keybase is so perfectly suited to making PGP encryption for e-mails easily available to the world, I was surprised that Keybase had yet to make more progress towards integrating Keybase with sending/receving encrypted e-mails. I would strongly suggest creating some sort of simple e-mail client application that can send and receive e-mails from a user's e-mail account of choice (e.g. gmail, yahoo mail, etc.). 

Basically, I want to be able to quickly find someone in Keybase in some sort of simple GUI application e-mail client running on my local computer, and compose an e-mail to that person which would automatically be encrypted. Also, when using that e-mail client application to view my e-mail Inbox, I want to be able to open up encrypted e-mails and have them automatically decrypted.
